k8s/demo/crunchy-postgres/README.md

```diff
@@ -1,35 +1,203 @@
+
+```
+test@Master:~/crunchy-postgres$ kubectl get statefulsets
+NAME      DESIRED   CURRENT   AGE
+pgset     2         2         15m
+
+test@Master:~/crunchy-postgres$ kubectl get pods
+NAME                                                             READY     STATUS    RESTARTS   AGE
+maya-apiserver-2245240594-ktfs2                                  1/1       Running   0          3h
+openebs-provisioner-4230626287-t8pn9                             1/1       Running   0          3h
+pgset-0                                                          1/1       Running   0          3m
+pgset-1                                                          1/1       Running   0          3m
+pvc-17e21bd3-c948-11e7-a157-000c298ff5fc-ctrl-3572426415-n8ctb   1/1       Running   0          3m
+pvc-17e21bd3-c948-11e7-a157-000c298ff5fc-rep-3113668378-9437w    1/1       Running   0          3m
+pvc-17e21bd3-c948-11e7-a157-000c298ff5fc-rep-3113668378-xnt12    1/1       Running   0          3m
+pvc-1e96a86b-c948-11e7-a157-000c298ff5fc-ctrl-2773298268-x3dlb   1/1       Running   0          3m
+pvc-1e96a86b-c948-11e7-a157-000c298ff5fc-rep-723453814-hpkw3     1/1       Running   0          3m
+pvc-1e96a86b-c948-11e7-a157-000c298ff5fc-rep-723453814-tpjqm     1/1       Running   0          3m
+
+test@Master:~/crunchy-postgres$ kubectl get svc
+NAME                                                CLUSTER-IP       EXTERNAL-IP   PORT(S)             AGE
+kubernetes                                          10.96.0.1        <none>        443/TCP             4h
+maya-apiserver-service                              10.98.249.191    <none>        5656/TCP            3h
+pgset                                               None             <none>        5432/TCP            14m
+pgset-primary                                       10.104.32.113    <none>        5432/TCP            14m
+pgset-replica                                       10.99.40.69      <none>        5432/TCP            14m
+pvc-17e21bd3-c948-11e7-a157-000c298ff5fc-ctrl-svc   10.111.243.121   <none>        3260/TCP,9501/TCP   14m
+pvc-1e96a86b-c948-11e7-a157-000c298ff5fc-ctrl-svc   10.102.138.94    <none>        3260/TCP,9501/TCP   13m
+
+
+test@Master:~/crunchy-postgres$ kubectl get clusterrolebinding permissive-binding
+NAME                 AGE
+permissive-binding   15m
+test@Master:~/crunchy-postgres$
 ```
-cd $HOME
-git clone https://github.com/openebs/openebs.git
-cd openebs/k8s/demo/crunchy-postgres
+
+Note: It may take some time for the pods to start as the images must be pulled and instantiated. This is also
+dependent on the network speed.
+
+
+## Verify Successful Crunchy-Postgres Deployment
+
+The verification procedure can be carried out using the following steps: 
+
+- Check cluster replication status between the Postgres primary and replica
+- Create a table in the default database as a Postgres user on the primary
+- Check data synchronization on the replica for created table
```

Comment:
Check data synchronization on the replica cluster in the table you have created.